lib/http_perform.c

```diff
@@ -161,14 +161,13 @@ sf_bool STDCALL http_perform(CURL *curl,
       1,      //base
       16      //cap
     };
-    /* TODO: let's remove this if not used.
+
     RETRY_CONTEXT retry_ctx = {
             0,      //retry_count
             network_timeout,
-            1,      // time to sleep
+            3,      // time to sleep
```

Comment:
- Should we combine `curl_retry_ctx` and `retry_ctx `
- We can combine line 160 to 170 and use `retry_ctx_init` instead.
- Is this change 1->3 to stay in sync with other client? If that's the case then we should update `retry_ctx_init` (since there we are initializing `sleep_time` with 1)
- Is the waiting time cap 16sec is what we're using in other clients?
- Finally it doesn't seem we are doing anything with `RETRY_CONTEXT->retry_timeout`, was there a purpose behind it?

Reply:
- Combined and used local variable 
- reverted to 1 (odbc uses 1 as start time)
- Changed the behavior to sleep in micro seconds not in seconds (feel like seconds is too huge for a retry, let me know if its otherwise)
- ODBC uses 16 as max time, 
- Using max number of retries, we can also use retry_timeout if thats better than the number of retries.